lib/terraforming/resource/ec2.rb

```diff
@@ -92,6 +93,10 @@ def in_vpc?(instance)
           (instance.subnet_id && instance.subnet_id != "" && instance.security_groups.length == 0)
       end
 
+      def monitoring_state(instance)
+        instance.monitoring.state == "enabled" ? 1 : 0
```

Comment:
`monitoring` can be `true` if `monitoring.state` is `enabled` __or__ `pending` :bulb: 

## REF
- [terraform/resource_aws_instance.go at 968ce8eea04b616a7d247c7bb1e49c5cd5cce684 · hashicorp/terraform](https://github.com/hashicorp/terraform/blob/968ce8eea04b616a7d247c7bb1e49c5cd5cce684/builtin/providers/aws/resource_aws_instance.go#L481)